appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -202,6 +209,8 @@ private void callDelegateWithOneError(
         fail();
       } catch (ApiProxyException exception) {
         assertEquals(expectedException.getClass(), exception.getClass());
+      } finally {
```

Comment:
I would prefer if we had only one finally to handle the logger reset (so `if(sync)` would be inside try ~ finally).